.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["orta.vscode-twoslash-queries", "rome.rome"]
```

Comment:
Since Rome has changed ownership and rebranded as Biome, there's the new extension: https://marketplace.visualstudio.com/items?itemName=biomejs.biome